### PR TITLE
Add random sorting

### PR DESCRIPTION
**Pull request by dannymichel**
Added random sorting to tv shows and movies lists
168119645-502f9949-7494-4ab2-bf1d-1fb05693f303

### CODE REVIEW

File: src/controllers/list.js [hunk 1]
```diff
@@ -1064,6 +1070,17 @@ class ItemsView {
         };
     }
 
+    getPlayCountSortOption() {
```
[the remark this answers is not in the document]
Comment (dannymichel) [reply src/controllers/list.js]: Changed in 8a2e2af2eb128ad6fc697315782449e8ab78c819

File: src/controllers/movies/movies.js [hunk 2]
```diff
@@ -197,6 +197,9 @@ import '../../elements/emby-itemscontainer/emby-itemscontainer';
                         items: [{
                             name: globalize.translate('Name'),
                             id: 'SortName,ProductionYear'
+                        }, {
+                            name: globalize.translate('Random'),
```
[the remark this answers is not in the document]
Comment (dannymichel) [reply src/controllers/movies/movies.js]: Changed in 8a2e2af2eb128ad6fc697315782449e8ab78c819

File: src/components/listview/listview.js [hunk 3]
```diff
@@ -42,6 +42,9 @@ import ServerConnections from '../ServerConnections';
 
             return name.toUpperCase();
         }
+        if (sortBy.indexOf('random') === 0) {
```
[the remark this answers is not in the document]
Comment (dannymichel) [reply src/components/listview/listview.js]: Changed in 8a2e2af2eb128ad6fc697315782449e8ab78c819

**Comment by dannymichel** [comment]
Made those changes 8a2e2af2eb128ad6fc697315782449e8ab78c819

**Review by thornbill — APPROVED** [review]

**Comment by Shirkit** [comment]
Was this feature removed? I don't see it on 10.9.0

**Comment by rightsaidfred99** [comment]
It's not available in Android TV client?

**Comment by lewgrant** [comment]
I used this feature a lot and after upgrading it has disappeared. Ideally the Home Screen would have the option of adding a "random" or "random unwatched" section like in Kodi but at least the random sort was an alternative.

Anyone know if this feature was removed for a reason or if maybe it's a regression?

**Pull request merged**